frontend/packages/console-app/src/components/lightspeed/Lightspeed.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react';
+import {
+  Alert,
+  Button,
+  Page,
+  PageSection,
+  Stack,
+  StackItem,
+  Title,
+  Tooltip,
+} from '@patternfly/react-core';
+import { WindowMinimizeIcon } from '@patternfly/react-icons';
+import { useTranslation } from 'react-i18next';
+import { useHideLightspeed } from '@console/app/src/components/user-preferences/lightspeed';
+import { k8sGetResource } from '@console/dynamic-plugin-sdk/src/utils/k8s';
+import { history } from '@console/internal/components/utils';
+import { ConsolePluginModel } from '@console/internal/models';
+import { FLAGS } from '@console/shared';
+import { useFlag } from '@console/shared/src/hooks/flag';
+import './Lightspeed.scss';
+
+const getLightspeedInstallationStatus = async () => {
+  try {
+    await k8sGetResource({
+      model: ConsolePluginModel,
+      name: 'lightspeed-console-plugin',
+    });
+    return true;
+  } catch (err) {
+    if (err.response.status !== 404) {
+      // eslint-disable-next-line no-console
+      console.log(err.message);
+    }
+    return false;
+  }
+};
+
+export const lightspeedOperatorURL =
+  '/operatorhub/all-namespaces?keyword=lightspeed&details-item=lightspeed-operator-redhat-operators-openshift-marketplace';
+
+const Lightspeed: React.FC = () => {
```

Comment:
Nit, this name could be more descriptive since the term Lightspeed is a bit overloaded. "LightspeedDiscovery"? No change needed in this PR though, I know we need to merge this.